### PR TITLE
registration page creation

### PR DESCRIPTION
**Pull request by ndefokou**
@stephane-segning registration page creation

### CODE REVIEW

File: power-pay-frontend/dev-dist/registerSW.js
Comment (Ngha-Boris) [inline power-pay-frontend/dev-dist/registerSW.js:1]: Hello @ndefokou what is this file doing in this PR please consider this.

File: power-pay-frontend/dev-dist/workbox-b5f7729d.js
Comment (Ngha-Boris) [inline power-pay-frontend/dev-dist/workbox-b5f7729d.js:1]: Hello @ndefokou what is this file doing in this PR please consider this.

File: power-pay-frontend/dev-dist/sw.js
Comment (Ngha-Boris) [inline power-pay-frontend/dev-dist/sw.js:1]: Hello @ndefokou what is this file doing in this PR please consider this.

**Review by Ngha-Boris — CHANGES_REQUESTED** [review]

**Comment by Ngha-Boris** [comment]
Please @ndefokou  also send a screenshot of the user registration front-end page so that we see how it looks like.

**Comment by ndefokou** [comment]
@Motouom done

File: power-pay-frontend/src/components/RegistrationForm.tsx
Comment (Motouom) [inline power-pay-frontend/src/components/RegistrationForm.tsx:1]: You can also remove the validation logic from here and also extract it into a separate component

**Review by Koufan-De-King — APPROVED** [review]
@Ngha-Boris 
I see the tests are now working. Looks good.

**Review by Koufan-De-King — CHANGES_REQUESTED** [review]
@ndefokou Consider the changes asked by @Ngha-Boris

**Comment by ndefokou** [comment]
@Koufan-De-King @Motouom done

**Comment by ndefokou** [comment]
Screenshot from 2024-04-18 17-38-53
Screenshot from 2024-04-18 17-38-43
Screenshot from 2024-04-18 17-38-06

**Comment by ndefokou** [comment]
@Ngha-Boris hier is it

**Review by Ngha-Boris — CHANGES_REQUESTED** [review]
Hello Mr @ndefokou please check the mock-up that Sinke sent in the Spring-jobs channel on discord and ensure that your mock-up is exactly like what  Sinke had sent that is for the part that concern you.

**Comment by Motouom** [comment]
> Screenshot from 2024-04-18 17-38-53 Screenshot from 2024-04-18 17-38-43 Screenshot from 2024-04-18 17-38-06

@ndefokou  This is your Mockup please make sure it looks the same. The only problem on the mockup is that the pin input field was forgotten. Never the less your output needs to match your mockup
image

**Comment by Motouom** [comment]
> Screenshot from 2024-04-19 09-39-41 Screenshot from 2024-04-19 09-39-30 @Ngha-Boris @Motouom hier is the correction

Awesome @ndefokou

**Comment by ndefokou** [comment]
@Ngha-Boris the correction is above

**Comment by Motouom** [comment]
@ndefokou  Please you should also add a Login button in the registration page such that if the user does have an account already he/she can just click there and it will take him to the login page.

The Login page has already been created so you just need to add the button and that is all

**Review by Koufan-De-King — APPROVED** [review]

**Comment by ndefokou** [comment]
Screenshot from 2024-04-22 09-32-23
@Motouom

**Review by Motouom — APPROVED** [review]

**Review by Koufan-De-King — APPROVED** [review]
Looks alright.

**Review by Ngha-Boris — APPROVED** [review]

**Review by Ngha-Boris — APPROVED** [review]

**Review by Ngha-Boris — CHANGES_REQUESTED** [review]
Please @ndefokou  ensure that all your commits are signed

**Review by Ngha-Boris — APPROVED** [review]

**Pull request merged**